exporter/keystoneexporter/exporter.go

```diff
@@ -37,16 +38,26 @@ func (k keystoneExporter) ConsumeMetrics(ctx context.Context, md pdata.Metrics)
 	return nil
 }
 
-func (k keystoneExporter) getMessage(md pdata.Metrics) (KsMessage, error) {
+func (k keystoneExporter) getMessage(md pdata.Metrics) (*KsMessage, error) {
 	events := make([]KsEvent, 0)
 
 	ocmds := internaldata.MetricsToOC(md)
 	k.log.Debug("constructing keystone message for metricsdata objects", zap.Int("count", len(ocmds)))
 	for _, ocmd := range ocmds {
 		k.log.Debug("constructing keystone message for metrics", zap.Int("count", len(ocmd.Metrics)))
 		for _, metric := range ocmd.Metrics {
-			if event, err := GetEvent(metric); err == nil {
-				events = append(events, event)
+			if metric.MetricDescriptor == nil {
```

Comment:
Did you want to also issue some self-metrics for this (whole PR)

Reply:
Good point.  Will add.

Reply:
Actually going to defer this as I don't see any part of the otel-collector producing metrics about itself.  A slightly larger project than this PR is intended to address.